Cargo.toml

```diff
@@ -22,3 +22,18 @@ unwrap_used = "warn"
 
 [profile.release]
 strip = true
+
+[workspace.dependencies]
```

Comment:
Dependabot (at least used to) break when declaring dependencies at the workspace level. I haven't tried it again recently though, so things could have changed. (And if it is still broken perhaps we can open another report upstream at https://github.com/dependabot/dependabot-core/issues ?)